agimus_demos_common/agimus_demos_common/launch_utils.py

```diff
@@ -97,6 +97,199 @@ def generate_include_franka_launch(launch_file_name: str) -> IncludeLaunchDescri
     )
 
 
+def generate_default_tiago_args() -> list[DeclareLaunchArgument]:
+    """Generates list of default arguments expected to be public interface of
+        launch files used by Agimus Demos for Tiago-Pro robot.
+
+    Returns:
+        list[DeclareLaunchArgument]: List of DeclareLaunchArgument objects with
+            arguments expected by `tiago_common.launch.py` and `tiago_common_lfc.launch.py`
+    """
+    return [
+        DeclareLaunchArgument(
+            "use_gazebo",
+            default_value="false",
+            description="Configures launch file for Gazebo simulation. "
+            + "If set to `true` launch file is configured for simulated robot. ",
+            choices=["true", "false"],
+        ),
+        DeclareLaunchArgument(
+            "use_rviz",
+            default_value="false",
+            description="Visualize the robot in RViz",
+            choices=["true", "false"],
+        ),
+        DeclareLaunchArgument(
+            "world_name",
+            default_value="empty",
+            description="Gazebo world name from the pal_gazebo_worlds package.",
+            choices=["empty", "pal_office"],
+        ),
+        DeclareLaunchArgument(
+            "base_type",
+            description="Base type.",
+            choices=["omni_base"],
```

Comment:
This is a naming for ROS frame id or what? Other parameters are separated with `-` while this one uses `_`. If there a reason for that?